tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh

```diff
@@ -64,7 +64,7 @@ for user in "${no_users[@]}"; do
 	log_mustnot zfs set userquota@$user=100m $QFS
 done
 
-log_note "can set all numberic id even that id is not existed"
+log_note "can set all numeric id even if that id is nonexistent"
```

Comment:
As I suggested in another PR, "can set all numeric id even if that id does not exist" might be more clear.

Reply:
Done!